### PR TITLE
Reformat tags for hotspot builds to ensure they match upstream tags

### PR DESCRIPTION
**Pull request by adamfarley**
*_adopt tags don't match anything upstream, so I'm removing the "_adopt" bit when we're running a hotspot build.

Testing here: https://ci.adoptium.net/job/build-scripts/job/openjdk21-pipeline/177/

### CODE REVIEW

**Comment by adamfarley** [comment]
Ok, typo caused a failure. Fixed the typo, and then decided that we don't want to put this logic within the build itself. Tidier if we put it in the pipeline job.

Change made. Running test now: https://ci.adoptium.net/job/build-scripts/job/openjdk21-pipeline/178/

**Comment by andrew-m-leonard** [comment]
run tests

**Comment by karianna** [comment]
Have pings the PMC on this one in case there's historical context

**Comment by gdams** [comment]
This won’t work, the whole reason for the _adopt tag is that it contains any Adoptium patches that have been merged from the dev branch into the release branch.

**Comment by andrew-m-leonard** [comment]
> This won’t work, the whole reason for the _adopt tag is that it contains any Adoptium patches that have been merged from the dev branch into the release branch.

@gdams for some context, this is "hotspot" builds, not "temurin" builds, see:
https://github.com/adoptium/ci-jenkins-pipelines/blob/8c0442b810fd456ff4027bc387c3059ce295ae9f/pipelines/jobs/configurations/jdk21.groovy#L24
We do a sanity build of upstream "hotspot" for jdk-21, which fails as obviously upstream openjdk don't have an _adopt tag..
If we think we should just not build this, then that's an option too? @sxa

**Comment by gdams** [comment]
Ahh okay sorry that makes sense now.

**Review by gdams — APPROVED** [review]

**Comment by sxa** [comment]
What is the problem that's being fixed here? That's not clear from the description. Is there something specific that's failing?

**Comment by adamfarley** [comment]
Heya Stewart. The problem we're fixing here is that the tag-specific ea builds of jdk21 ([example](https://ci.adoptium.net/job/build-scripts/job/jobs/job/jdk21/job/jdk21-linux-aarch64-hotspot/61/)) are failing when building the hostpot (NOT Temurin) variant, because we're seeking a specific tagged level which IS present at Adoptium, but is NOT present when we clone directly from the upstream OpenJDK source repos.

In short, we need to remove the "_adopt" bit from hotspot-variant builds before they will work.

This change does that in a way that lets us continue to execute the hotspot builds with the same pipeline job as the Temurin builds.

**Comment by adamfarley** [comment]
Looks like we have no remaining objections. Will squash and merge.

**Comment by sxa** [comment]
> Heya Stewart. The problem we're fixing here is that the tag-specific ea builds of jdk21 ([example](https://ci.adoptium.net/job/build-scripts/job/jobs/job/jdk21/job/jdk21-linux-aarch64-hotspot/61/)) are failing when building the hotspot (NOT Temurin) variant

Makes sense - thanks for clarifying.
